src/components/Modal.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react"
+import Button from "react-bootstrap/Button"
+import Modal from "react-bootstrap/Modal"
+
+function Example() {
```

Comment:
I would rethink the name of this component. And make sure you are following the function syntax of your other components, `const ComponentName = () => {`